src/parser.c

```diff
@@ -328,28 +381,47 @@ enum parse_rv parse_Output(struct Output_state *const state, parser_meta_state_t
         case 0:
             CALL_SUBPARSER(uint32State, uint32_t);
             state->type = state->uint32State.val;
+            PRINTF("Output Type: %d\n", state->type);
             state->state++;
             switch (state->type) {
                 default: REJECT("Unrecognized ouput type");
                 case 0x00000007:
                     INIT_SUBPARSER(secp256k1TransferOutput, SECP256K1TransferOutput);
+                    break;
+                case 0x00000016:
+                    if(state->noStakeableLock) REJECT("Can't parse nested StakeableLockOutput");
+                    INIT_SUBPARSER(stakeableLockOutput, StakeableLockOutput);
+                    break;
             }
         case 1:
             switch (state->type) {
                 default: REJECT("Unrecognized ouput type");
```

Comment:
"output". Same above

Reply:
Huh. Not actually part of the diff, looks like; regardless, might as well fix.